.eslintrc.js

```diff
@@ -236,7 +213,7 @@ module.exports = {
       {
         selector: "CallExpression[callee.property.name='strictEqual'][arguments.0.type='Literal']:not([arguments.1.type='Literal']):not([arguments.1.type='ObjectExpression']):not([arguments.1.type='ArrayExpression']):not([arguments.1.type='UnaryExpression'])",
         message: 'The first argument should be the `actual`, not the `expected` value.',
-      }
+      },
```

Comment:
If you want to have a consistent style, what about enabling the rule at the top of the file? That way it would be possible to slowly migrate some files.